core/src/components/textarea/textarea.scss

```diff
@@ -29,18 +37,27 @@
   --placeholder-opacity: #{$placeholder-opacity};
   --padding-top: 0;
   --padding-end: 0;
-  --padding-bottom: 0;
+  --padding-bottom: #{$textarea-padding-bottom};
```

Comment:
This should apply to modern textareas only otherwise it's a breaking change to the legacy template